assets/app/scripts/services/imageStreamResolver.js

```diff
@@ -27,8 +31,10 @@ angular.module('openshiftConsole')
         }
 
         var imageStreamImagePromise = DataService.get("imagestreamimages", imageStreamImageRef, context);
-        imageStreamImagePromise.then(function(image) {
-          imagesByDockerReference[dockerRef] = image;
+        imageStreamImagePromise.then(function(imageStreamImage) {
+          if (imageStreamImage) {
+            imagesByDockerReference[dockerRef] = imageStreamImage.image;
```

Comment:
the parent object can be present, and the `image` attribute could be null/undefined. Do we still want it in the map in that case?